### DISCUSSION TITLE
MQTT Telegram on value change, greater than defined threshold

### DISCUSSION THREAD

**Discussion by KasiBurnsRed** [arendst/Tasmota#12327] [Q&A] [answered]
Hello, i got two Gosund SP1 with Tasmota 9.2.0.
Actually the telemetry period is set to 10 (i think that means 10 seconds).
Is there a possibility to set a trigger, that the telegrams where send triggerd by a change of a value greater than a defined threshold?
For example: i only want to get a new MQTT telegram if the value of power changed by more than 50W compared to the previous one.
Thanks.

**Comment by Jason2866** [comment, marked as answer]
See https://tasmota.github.io/docs/Rules/#adjust-powerdelta-according-to-current-power-values

**Reply by sfromis** [reply]
You do not even need rules to adjust PowerDelta. Just configure it once and for all, and then have triggers based on the JSON published. `PowerDelta 150` would match the use case of a change of 50W.
